.github/workflows/publish.yaml

```diff
@@ -0,0 +1,79 @@
+name: Docker Image Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  EDGE: v1.14/alpine
+
+jobs:
+  publish:
+    name: Publish Docker Image
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [v1.14]
```

Comment:
It feels weird to have a matrix for a single container. I'd prefer to keep it simple until we actually get to the point of wanting other containers. I'd also move all the container support to the root of the repo, no need to bury it until we actually need to plus people can just do a `git clone ... && docker build .` to get it going.

Reply:
I personally think it's better to start with the expected structure as it has implications that might be missed by a naïve "update" to add another OS; see the release publish step that must only run once.